release-notes/README.md

```diff
@@ -4,6 +4,12 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## March 31, 2022
+
+### iOS AEPAnalytics 3.0.4
+
+* Fixed `getTrackingIdentifier` and `getVisitorIdentifier` APIs to `return nil` and not `AEPError.unexpected` error when AID/VID values are not found in persistence.
```

Comment:
how about change and not `AEPError.unexpected` error to
but not `AEPError.unexpected` error

Reply:
or we can use "instead of AEPError.unexpected ..." - same below